starknet_py/compile/compiler.py

```diff
@@ -62,17 +61,6 @@ def compile_contract(self) -> str:
         )
 
 
-def create_contract_class(
```

Comment:
Is it ok to remove it from here? Can't we just leave this and add the new one somewhere?

Reply:
Right, these should be duplicated for now.